doc/ref_impl/cntt-ri2/chapters/chapter01.md

```diff
@@ -55,12 +55,13 @@ The scope of this document is as follows:
 
 The main communities involved in driving requirements and development of this RI are:
 
-- **CNTT RC2 workstream**
-  - The **RC2 workstream** within CNTT takes requirements from the RM and RA2 and identifies test cases and a test integration suite that will be used to verify and validate conformance of an implementation. RI2 will be a deployable reference for operators, vendors, and others to develop against.
+- **Anuket RC2 workstream**
+  - The **RC2 workstream** within Anuket takes requirements from the RM and RA2 and identifies test cases and a test integration suite that will be used to verify and validate conformance of an implementation. RI2 will be a deployable reference for operators, vendors, and others to develop against.
 
-- **OPNFV**
-  - The **Kuberef** project in OPNFV acts as the main integration point for RI2: Its purpose it to provide all necessary glue code and mechanisms for i) integrating and configuring selected upstream Kubernetes deployers and components into a deployable platform, ii) continuously deploy the integrated platform in OPNFV labs and iii) run the latest RC2 test suite against the deployed platform. The test results are meant to be fed back to RA2 and RC2 to support the refinement of architecture requirements and the selection of test cases for the RC2 test suite.
+- **Anuket Kuberef**
+  - The **Kuberef** project in Anuket acts as the main integration point for RI2: Its purpose it to provide all necessary glue code and mechanisms for i) integrating and configuring selected upstream Kubernetes deployers and components into a deployable platform, ii) continuously deploy the integrated platform in OPNFV labs and iii) run the latest RC2 test suite against the deployed platform. The test results are meant to be fed back to RA2 and RC2 to support the refinement of architecture requirements and the selection of test cases for the RC2 test suite.
```

Comment:
OPNFV labs >Anuket labs

Reply:
Thanks for pointing out - I didn't change the labs part as I wasn't sure if Anuket Labs is the new name for former OPNFV labs - so far the lab related services like jira, gerrit, etc. still seem to be tied to OPNFV. Nevertheless, I'll rename it to Anuket Labs.